src/components/messageSpace/messageSpace.stories.tsx

```diff
@@ -14,11 +14,13 @@ import {
   StreamingText,
   Table,
   Text,
+  type ThreadableMessage,
   Video,
   YoutubeVideo,
 } from '..'
 import CodeSnippet from '../codeSnippet/codeSnippet'
 import Icon from '../icon'
+import Copy from '../messageCanvas/actions/copy/copy'
 import MessageSpace from './messageSpace'
 
 export default {
```

Comment:
Adding description here for each field is not required if we use JsDoc Comments. We can also avoid duplicate things if we define an interface for the shared props between MessageSpace and MessageCanvas, for example, 

```typescript
export interface MessageContainerProps {
  /** Profile icon to be shown before sender's name. */
  getProfileComponent?: (message: ThreadableMessage) => ReactNode
  /** A function that returns a single React element which may be composed of several actions supported for the message, such as editing, copying, and deleting, etc.
   * In case no actions are applicable or available for a particular message, the function may return `undefined`.
   * This approach offers flexibility in tailoring message interactions to specific application requirements.
   * To define individual message actions, developers can extend the `Action` component's functionality.
   * One such example is the CopyText component.
   */
  getActionsComponent?: (message: ThreadableMessage) => ReactNode | undefined
}

export interface MessageCanvasProps extends MessageContainerProps {
  /** Message information to be displayed. Please see the `MessageSpace` docs for more information about the `ThreadableMessage` interface. */
  message: ThreadableMessage
  /** React component to be displayed in the message canvas. */
  children: ReactNode
}

```

Reply:
I will extend `MessageContainerProps`, but we still need to provide details about `supportedElements` and `messages` here to explain `ComponentMap` and `ThreadableMessage`